docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -4,33 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>I have some doubts about the idea that "2. Create a Generalized URL" should be a standalone step. A "Generalized URL" is not actually a formal concept in Pangeo Forge. Really what you are creating here is a format string for the url. I would prefer to eliminate 2 and fold it in to 1 as part of "understanding the file pattern"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
So that part was intentional on my part. I was thinking that it should be its own step to really emphasize that understanding the structure of the dataset is a really key first step.

You suggest using URL Template String in another comment, for the change I combined "understand dataset" and "create url template string" into one point, but kept both phrases in the name of the step. I also de-capitalized generalized URL and changed it to URL Template String where appropriate.